RevenueCatUI/Templates/V2/ViewHelpers/ProgressViewModifier.swift

```diff
@@ -40,7 +42,7 @@ struct ProgressViewModifier: ViewModifier {
                 .progressViewStyle(CircularProgressViewStyle(tint: bestTint(for: colorInfo)))
         case .image, .none:
             ProgressView()
-                .progressViewStyle(CircularProgressViewStyle(tint: .white))
+                .progressViewStyle(CircularProgressViewStyle(tint: .gray))
```

Comment:
Just curious: this change, is it to make it visible in both light and dark mode?

Reply:
Yeah yeah, it would make it visible for both the default-ish feeling white and black backgrounds that can change for light and dark mode 😁 I noticed it on a few paywalls I was playing with. I think there are better odds are that defaulting to gray will show up better most of the time 🎲 😇